client/reports.go

```diff
@@ -319,6 +319,7 @@ func (c *Client) staticServerSync(gcas GCAServer, gcasKey glow.PublicKey, gcaKey
 	}
 
 	c.EventLog.Printf("sync successful with %v", gcas.Location)
+	c.updateSyncFile()
```

Comment:
you should check the error here and create an event log that contains the error if there's an issue